composer.json

```diff
@@ -5,18 +5,18 @@
   "license": "proprietary",
   "require": {
     "php": ">=7.1",
-    "spryker/cart": "^5.0.0",
+    "spryker/cart": "^5.0.0 || ^7.0.0",
```

Comment:
The Client Function    
`public function addValidItems(CartChangeTransfer $cartChangeTransfer, array $params = []): QuoteTransfer` was introduced only in Cart 5.1.0